src/experimental/safe_html_transform/transforms/exp_safe_html.py

```diff
@@ -293,24 +293,22 @@ def convert(self, orig, data, **kwargs):
         else:
             # append html tag to create a dummy parent for the tree
             html = "<html>%s</html>" % orig
+            parser = etree.HTMLParser()
+            tree = etree.parse(StringIO(html), parser)
+            for element in tree.getiterator():
+                if(element.tag == 'h3' or element.tag == 'h4' or element.tag == 'h5' or element.tag == 'h6' or element.tag == 'div'):
```

Comment:
@prakharjoshi small improvement: "element.tag in ['h3', 'h4', ...]" is more readable.